app/__init__.py

```diff
@@ -1,5 +1,7 @@
 from flask import Flask
+from app.config import Config
```

Comment:
Config things will need to be sorted in a merge - I think keep whatever is currently in the main branch, assuming that still works for you